### PR TITLE
MAGEDOC-3207: Add PHP 7.1 support to table

### PR DESCRIPTION
**Pull request by shrielenee**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add PHP 7.1 support for 2.1.16+ to the PHP table in the [system-requirements-tech.md file/page](https://devdocs.magento.com/guides/v2.1/install-gde/system-requirements-tech.html).

**Should not be merged until 2.2.7 release.**

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.1/install-gde/system-requirements-tech.html
- https://devdocs.magento.com/guides/v2.1/install-gde/prereq/php-centos.html (uses include)
- https://devdocs.magento.com/guides/v2.1/install-gde/prereq/php-ubuntu.html (uses include)

In conjunction with another open PR, #3230, that adds a note that for 2.1.16 and later, Magento supports PHP 7.1.

### CODE REVIEW

File: _includes/install/php_2.1.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-| 5.5.x | 5.6.0&ndash;5.6.4 | 5.6.5&ndash;5.6.x | 7.0.0, 7.0.1 | 7.0.2 | 7.0.3 | 7.0.4 | 7.0.5 | 7.0.6&ndash;7.0.x | 7.1.x |
+| 5.5.x | 5.6.0&ndash;5.6.4 | 5.6.5&ndash;5.6.x | 7.0.0, 7.0.1 | 7.0.2 | 7.0.3 | 7.0.4 | 7.0.5 | 7.0.6&ndash;7.0.x | 7.1.16+ |
```
Comment (jeff-matthews) [inline _includes/install/php_2.1.md]: I din't think this is right... Was this a typo?

Comment (shrielenee) [reply _includes/install/php_2.1.md]: All fixed up.

**Comment by shrielenee** [comment]
Can you also take a look at the companion PR? https://github.com/magento/devdocs/pull/3230

**Comment by magento-cicd2** [comment]
Can one of the admins verify this patch?

**Pull request merged**